tests/unit/moduleapi/cluster.tcl

```diff
@@ -5,6 +5,25 @@ source tests/support/cli.tcl
 # cluster creation is complicated with TLS, and the current tests don't really need that coverage
 tags {tls:skip external:skip cluster modules} {
 
+set testmodule [file normalize tests/modules/cluster.so]
+set modules [list loadmodule $testmodule]
+start_cluster 3 0 [list config_lines $modules] {
+    set node1 [srv 0 client]
+    set node2 [srv -1 client]
+    set node3 [srv -2 client]
+
+    test "Cluster module send message API - VM_SendClusterMessage" {
+        assert_equal OK [$node1 test.pingall]
+        assert_equal 2 [CI 0 cluster_stats_messages_module_sent]
+        wait_for_condition 50 100 {
+            [CI 1 cluster_stats_messages_module_received] eq 1 &&
```

Comment:
We don't check that the message receiver callback was called. Should we?

An idea is to let `test.pingall` be a blocking command that waits for the pong from each node and returns the number of replies, but maybe that's more complicated. You can decide.

Reply:
I think it's beyond the scope of this PR.